app/code/core/Mage/Customer/Block/Account/Dashboard/Info.php

```diff
@@ -34,11 +34,22 @@
 
 class Mage_Customer_Block_Account_Dashboard_Info extends Mage_Core_Block_Template
 {
+    /**
+     * @var Mage_Newsletter_Model_Subscriber
+     */
+    public $_subscription;
```

Comment:
Suggestion: make class attribute private

There's no need to make this attribute accessible from outside (if so, please reference)